pkg/okteto/preview.go

```diff
@@ -107,8 +107,8 @@ type previewEnv struct {
 }
 
 type deployPreviewResponse struct {
-	Action  actionStruct
-	Preview previewIDStruct
+	Id     graphql.String
```

Comment:
I thinks this is not the same and it's a breaking change

We are changing this response:

```
{
    "preview": {
         "id": ...,
     },
     "action": {
           .....
       }
}
```

by this:

```
{
    "id": ....,
    "action": {
         .....
     }
}
```

Reply:
We are, but the return value from `deployPreview` has always been:

```
{
  "id": "my-id",
  "action": {
    ...
  },
  "preview": {
    "id": "my-id"
    ...
  }
}
```

So the top level `id` will exist in **any** okteto backend version

Reply:
Oh, ok, yeah, I didn't check the preview response shape. It seems it's the same